bundles/admin/javascript-hooks/header.rst

```diff
@@ -0,0 +1,110 @@
+Header
+======
+
+The header-hook renders a header (the blue bar with the toolbar in it) into your component.
+Moreover it takes care of the tab-handling.
+
+Most often you'll find yourself defining a component which contains the header-hook and
+methods for manipulating the toolbar or other methods which implement behaviour which is
+the same over all tabs.
+
+The header-hook in such a component could look something like:
+
+.. code-block:: javascript
+
+    header: {
+       tabs: {
+           url: 'url/to/tabsData'
+       },
+       toolbar: {
+           languageChanger: true
+           buttons: {
+               save: {},
+               settings: {
+                   options: {
+                       dropdownItems: {
+                           delete: {}
+                       }
+                   }
+               }
+           }
+       }
+       title: 'My title',
+    }
+
+What this hook does is essentially the following:
+
+* It initializes the tabs with the data returned from the defined url. From then on
+  you'll never have to worry about tabs again. The whole routing to and starting your
+  tabs component is handled automatically if the data returned by ``tabs.url`` has
+  the right format.
+* It initializes the toolbar with the defined buttons. For information on how to configure buttons
+  and even how to create your own buttons have look :doc:`here </bundles/admin/sulu-buttons>`.
+* It injects the title into every tab (or into your current component if no tabs specified)
+
+.. note::
+    The header-hook can also be a function which returns the object seen in the example. Within this function
+    you have access on things like ``this.options``
+
+
+List of all available options:
+------------------------------
+
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
+      - Type
+      - Description
+    * - title
+      - String|Function
+      - A title which gets injected into every tab-component or into the current component if no tabs exist. If it's
+        a function it must return a string.
+    * - noBack
+      - Boolean
+      - if true no back icon gets rendered
+    * - tabs
+      - Object
+      - contains configuration-properties about the tabs
+    * - tabs.url
+      - String
+      - url to load the tabs-data from
+    * - tabs.data
+      - Object
+      - tabs-data. Either this option or the ``tabs.url`` option must be set when working with tabs.
```

Comment:
@danrot do you mean the description in the documentation or the whole functionality?